app/assets/stylesheets/fae/modules/forms/_label.scss

```diff
@@ -18,4 +18,19 @@ label {
     font-style: italic;
     line-height: 1.3;
   }
+
+  &.label--two_col {
+    display: flex;
```

Comment:
Did some checking with @jamesmk and it looks like current support is at IE >= 11